### DISCUSSION TITLE
SX1262 Module

### DISCUSSION THREAD

**Discussion by polentacremosa** [LoRaMesher/LoRaMesher#59] [General]
Hi @Jaimi5 !

Thank you again for the library and the support.
I'm trying to use the library with the module SX1262 on a LilyGo T3S3 V1.2 with no success.

After configuring the code below, I got the error "Radio module gave error: -2", which RadioLib describes as "RADIOLIB_ERR_CHIP_NOT_FOUND   (-2)
  | Radio chip was not found during initialization. This can be caused by specifying the wrong chip type in the constructor (i.e. calling SX1272 constructor for SX1278 chip) or by a fault in your wiring (incorrect slave select pin).

Do you know what it could be, or what else has to be adapted to the code to work?

Configuration:
```

    //Set the configuration of the LoRaMesher for LILYGO Lora32 V2 1.6
    config.loraCs = 7;
    config.loraRst = 8;
    config.loraIrq = 34; //DIO0
    config.loraIo1 = 33; 

config.module = LoraMesher::LoraModules::SX1262_MOD;
```

**Comment by Jaimi5** [comment]
With a first look it seems something about the pins, have you tried swapping IRQ and IO1?

**Reply by polentacremosa** [reply]
Yes, I tried but it didn't work.

**Reply by polentacremosa** [reply]
Have you been able to run the library on an SX1262 board, and if so, could you share the code?

Thank you!
